src/services/schema.tsx

```diff
@@ -84,7 +91,7 @@ function getEdges<ReqFields extends string = string>(node: GeneralRecordType<Req
 function getEdges<ReqFields extends string = string>(node: GeneralRecordType<ReqFields> | null): string[];
 
 function getEdges<ReqFields extends string = string>(node: GeneralRecordType<ReqFields> | null = null) {
-  const list: string[] = schemaDefn.schema.E.subclasses.slice().map((classModel) => classModel.name);
+  const list: string[] = schemaDefn.children('E');
```

Comment:
this differs slightly in that before it was returning a copy and now its not. it get the old behaviour the `.slice()` should be added back